### PR TITLE
GDB-6782 Implement cluster configuration edit action

### PR DESCRIPTION
**Pull request by yordanalexandrov**
- added button to go to edit cluster view, which is only visible to leader node
 - refactored create cluster view to provide edit functionality
 - refactored CreateClusterCtrl to host update cluster functionality for less code duplication, renamed to CreateUpdateClusterCtrl

### CODE REVIEW

File: src/pages/cluster-management/cluster-configuration.html [hunk 1]
```diff
@@ -4,6 +4,13 @@
     <h1>{{pageTitle}}</h1>
     <div onto-loader-new ng-if="loader" class="cluster-loader" message="getLoaderMessage" size="75"></div>
 
+    <div ng-if="!loader && noClusterExists">
+        <p class="alert alert-danger" ng-show="noClusterExists">
```
Comment (svilenvelikov) [inline src/pages/cluster-management/cluster-configuration.html:8]: Double check for noClusterExists. Do you need both?

**Review by svilenvelikov — APPROVED** [review]

File: src/pages/cluster-management/cluster-configuration.html (hunk 1 above)
Comment (yordanalexandrov) [reply src/pages/cluster-management/cluster-configuration.html]: Not any more. Removed it.

**Review by svilenvelikov — APPROVED** [review]

**Comment by yordanalexandrov** [comment]
Obsolete

**Comment by yordanalexandrov** [comment]
> Obsolete

Rafactored

**Comment by yordanalexandrov** [comment]
As the changes are way too different I decided to close this for good and open new PR

**Pull request closed without merging**